cmd/protoc-gen-go-tetragon/eventchecker/multichecker.go

```diff
@@ -140,33 +140,38 @@ func generateUnorderedEventChecker(g *protogen.GeneratedFile) error {
             return true, nil
         }
 
+        totalMatched := checker.totalChecks - pending
         if logger != nil {
-            logger.Infof("UnorderedEventChecker: %d/%d matched", checker.totalChecks-pending, checker.totalChecks)
+            logger.Infof("UnorderedEventChecker: starting checks with %d/%d total matched", totalMatched, checker.totalChecks)
```

Comment:
To me "starting checks" here would imply the checker is starting for the first time, not running checks on a specific event. Maybe consider rewording this somehow?

Reply:
How about "starting event check(s)"?

Reply:
@kkourt maybe "checking event"?